engines/scumm/script_v6.cpp

```diff
@@ -495,6 +495,79 @@ void ScummEngine_v6::o6_pushByteVar() {
 }
 
 void ScummEngine_v6::o6_pushWordVar() {
+// BACKYARD BASEBALL 2001 ONLINE CHANGES
+#if defined(USE_ENET) && defined(USE_LIBCURL)
+	if (ConfMan.getBool("enable_competitive_mods")) {
+		// Sprinting in competitive Backyard Baseball is considered too weak in its current state. This will increase how effective
+		// it is, limiting the highest speed characters enough to where they cannot go TOO fast.
+		if (_game.id == GID_BASEBALL2001 && _currentRoom == 3 && vm.slot[_currentScript].number == 2095 && readVar(399) == 1) {
+			int offset = _scriptPointer - _scriptOrgPointer;
+			int sprintCounter = readArray(344, vm.localvar[_currentScript][0], 1);
+			int sprintGain = vm.localvar[_currentScript][4];
+			if (offset == 42273) {
+				if (sprintCounter >= 11) {
+					if (vm.localvar[_currentScript][5] == 12 || vm.localvar[_currentScript][5] == 11 || vm.localvar[_currentScript][5] == 10) {
```

Comment:
`vm.localvar[_currentScript][5]` is used 7 times in this block. Consider assigning it to a variable, to improve readability